amivapi/events/model.py

```diff
@@ -436,6 +436,12 @@
                 'json_schema': True,
                 'only_if_not_null': 'spots',
             },
+            'moderator_signup_text': {
```

Comment:
I think the naming of the field is not accurate. It does not matter whether there is a moderator set for the event.

I would rename the field to something like `signup_additional_info`.
Other suggestions for the field name are welcome as I am not sure that this is the best name.